README.md

```diff
@@ -136,7 +139,7 @@ To configure `index-provider` to expose the delegated routing server, use the fo
 "DelegatedRouting": {
   "ListenMultiaddr": "/ip4/0.0.0.0/tcp/50617",
   "ProviderID": "PEER ID OF YOUR IPFS NODE",
-  "Addrs": // List of multiaddresses that you'd like to be advertised to IPNI. If not specified, Swarm addresses of the Kubo node will be used.
+  "Addrs": [] // List of multiaddresses that you'd like to be advertised to IPNI
```

Comment:
@jtsmedley would specifying Announce addrs here instead of Kubo solve the problem you had in https://github.com/ipfs/kubo/issues/9823?

Reply:
Thank you. Adding.